### PR TITLE
release 5.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by mmerfort — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
...
+- Drop CentOS 6 support [\#566](https://github.com/voxpupuli/puppet-python/pull/566) ([bastelfreak](https://github.com/bastelfreak))
+- Drop EOL Debian 8 [\#563](https://github.com/voxpupuli/puppet-python/pull/563) ([bastelfreak](https://github.com/bastelfreak))
+
+**Fixed bugs:**
+
+- Fix undefined method error in facts [\#555](https://github.com/voxpupuli/puppet-python/pull/555) ([wiebe](https://github.com/wiebe))
+
+**Closed issues:**
+
+- add pip2 to Enum for pip\_provider [\#567](https://github.com/voxpupuli/puppet-python/issues/567)
```
Comment (igalic) [inline CHANGELOG.md:22]: this is misleading, as it makes it look we did, as the title suggests

**Review by igalic — CHANGES_REQUESTED** [review]
do we have a wontfix label?

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md:22]: added the `skip-changelog` label to that issue and regenerated the changelog

**Review by igalic — APPROVED** [review]
okay, i guess 🤷🏻‍♀️

**Pull request merged**